server/server.go

```diff
@@ -233,15 +250,27 @@ func New(config *Config) (*Server, error) {
 
 	utils.StartReaper()
 
-	if err := os.MkdirAll(config.ImageDir, 0755); err != nil {
+	store, err := storage.GetStore(storage.StoreOptions{
+		RunRoot:            config.RunRoot,
+		GraphRoot:          config.Root,
+		GraphDriverName:    config.Storage,
+		GraphDriverOptions: config.StorageOption,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	imageService, err := scri.GetStorageImageService(store, config.DefaultTransport)
```

Comment:
Okay, just pushed changes which I think move us closer.  We still pull in kube by way of containers/image's support for openshift, but we're insulated from changes in the runtime API, at least.